recipe/meta.yaml

```diff
@@ -17,17 +17,16 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3.7
     - pip
     - wheel
     - setuptools >=30.0.0
   run:
-    - python >=3.6
-    # mccabe 0.7.0 is a noarch python package now and it hasn't broken changes for flake8
-    - mccabe >=0.6.0,<0.8.0
+    - python >=3.7
```

Comment:
This Python restriction is not correct; the [upstream setup.cfg](https://gitlab.com/pycqa/flake8/-/blob/4.0.1/setup.cfg#L23-27) suggests that support goes back at least as far back as Python 3.6.

Reply:
https://github.com/AnacondaRecipes/flake8-feedstock/blob/2515774498f3f67b9fa7e84a9378f106f46f5349/recipe/meta.yaml#L25

The python pinnings were corrected